src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -198,7 +198,10 @@ internal class CommandLineParameterParser
 
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/demos/DSC/readme.md.https://docs.microsoft.com/powershell/dsc/overview should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```

Reply:
@TravisEz13 Shall I replace this URL to `https://docs.microsoft.com/powershell/dsc`?